test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -3138,6 +3122,35 @@ def forward(self, x):
         x = torch.randn(2, 5)
         self.run_test(ShapeModule(), (x,), rtol=1e-3, atol=1e-5)
 
+    @skipIfUnsupportedMinOpsetVersion(12)
```

Comment:
ONNX Celu only takes float inputs. How about pytorch? Have you tested this on other input data types?

Reply:
I think alpha is float in pytorch too https://github.com/pytorch/pytorch/blob/master/torch/nn/functional.py#L1251
I added one test where alpha is int

Reply:
No I'm not asking about alpha, I'm asking about inputs.

Reply:
You can create a test for an int input using torch.randint()

Reply:
I think pytorch CCELU only accepts foalt inputs, tried with using int inputs (torch.randint()) and got the following:
RuntimeError: "elu_cpu" not implemented for 'Long'